### PR TITLE
Front websocket message

### PR DESCRIPTION
**Pull request by Mamar-dev**
Add websocket management and event dispatcher. 

Automaticaly update charge points status in the charge point list page

### CODE REVIEW

File: src/front/src/Home.tsx [hunk 1]
```diff
@@ -1,9 +1,65 @@
...
+    websocket: WebSocket | undefined;
+
+    constructor(){
+      super()
+      this.connected = false;
+    }
+
+    // Start 
+    startWebSocket(): void {
+      if(this.connected){
```
Comment (tvelu77) [inline src/front/src/Home.tsx:24]: An if should be written like this :
``` if (this.connected) {```
Don't forget spacing

File: src/front/src/pages/BornesTable.tsx [hunk 2]
```diff
@@ -53,10 +72,11 @@ export function ChargePointTable() {
       error: error,
       onSelection: chargePoint => { console.log("Selected item : " + chargePoint.id) },
       formatter: (chargePoint) => {
+        let color = chargePoint.status.state ? '#28FD41' : '#FF7552'
```
Comment (tvelu77) [inline src/front/src/pages/BornesTable.tsx:75]: Could be defined as a const ?

File: src/main/java/fr/uge/chargepointconfiguration/chargepoint/notification/ChargePointWebsocketNotification.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * Charge point update websocket notification.
+ */
+public record ChargePointWebsocketNotification(int id, Status status)
+        implements WebSocketNotification {
+
+  /**
+   * Transform a charge point to the charge point update notification.
+   *
+   * @param chargepoint updated charge point.
```
Comment (tvelu77) [inline src/main/java/fr/uge/chargepointconfiguration/chargepoint/notification/ChargePointWebsocketNotification.java:15]: Don't forget, you can do ```{@link Chargepoint}.``` for the doc

**Review by tvelu77 — APPROVED** [review]

File: src/front/src/Home.tsx (hunk 1 above)
Comment (Mamar-dev) [reply src/front/src/Home.tsx:24]: Non

File: src/front/src/pages/BornesTable.tsx (hunk 2 above)
Comment (Mamar-dev) [reply src/front/src/pages/BornesTable.tsx:75]: Not important in that case.

**Pull request merged**